opentelemetry/pom.xml

```diff
@@ -0,0 +1,36 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <parent>
+        <groupId>io.quarkus.ts.qe</groupId>
+        <artifactId>parent</artifactId>
+        <version>1.0.0-SNAPSHOT</version>
+        <relativePath>..</relativePath>
```

Comment:
@kshpak @Sgitario 
WDYT if we move on all the monitoring modules (as this one) to a "monitoring" folder as security?

Reply:
+1

Reply:
module moved under /security/monitoring/opentelemetry
profile micrometer was renamed into monitoring
@Sgitario wdyt?

Reply:
Not under `security`, simply `monitoring`.
Pablo's point was that the same we have the folder `security` with the modules `keycloak`, `keycloak-webapp`, ... We should have a `monitoring` folder to group all the modules related to monitoring. At the moment, we already have a module `microprofile` that should be moved to the new `monitoring` folder.

Reply:
Changed. Separate commit for microprofile module migration done